Esri/arcgis-runtime-samples-ios
# [Update] Replace instances of `AGSServiceFeatureTable.applyEdits` , 1st PR

## Description

This PR replaces various usage of `applyEdits` method on `AGSServiceFeatureTable` to `AGSServiceGeodatabase`, to prevent data inconsistency issue.

I'll break down the Epic into 2 PRs to make it easier to review. This is the 1st PR.

All the behaviors should remain the same. Coding style has been updated/refactored to follow our latest style guide. Also, a few UI related bugs are fixed. But we'll still request Sarat's review to ensure nothing is breaking the tests.

Also, all comment strings are formatted.

Finally, READMEs will be updated in yet another separate PR, after we finish updating the common designs.

## Linked Issue(s)

`common-samples/issues/3371`

## How To Test

Go though all changed samples and see if they work as expected.

<details><summary>Example change</summary>

```swift
/// Load and set a service geodatabase from a feature service URL.
/// - Parameter serviceURL: The URL to the feature service.
func loadServiceGeodatabase(from serviceURL: URL) {
    let serviceGeodatabase = AGSServiceGeodatabase(url: serviceURL)
    serviceGeodatabase.load { [weak self] error in
        guard let self = self else { return }
        if let error = error {
            self.presentAlert(error: error)
        } else {
            let featureTable = serviceGeodatabase.table(withLayerID: 0)!
            self.featureTable = featureTable
            self.serviceGeodatabase = serviceGeodatabase
            // Add the feature layer to the operational layers on map.
            let featureLayer = AGSFeatureLayer(featureTable: featureTable)
            self.mapView.map?.operationalLayers.add(featureLayer)
        }
    }
}

// then...
serviceGeodatabase.applyEdits { [weak self] (featureTableEditResults: [AGSFeatureTableEditResult]?, error: Error?) in
    if let error = error {
        self?.presentAlert(message: "Error while applying edits:\(error.localizedDescription)")
    } else {
        if let featureTableEditResults = featureTableEditResults,
           featureTableEditResults.first?.editResults.first?.completedWithErrors == false {
            self?.presentAlert(message: "Edits applied successfully")
        }
    }
}
```

</details>
